api/api/openapi.bundle.yaml

```diff
@@ -2299,8 +2304,12 @@ components:
           items:
             $ref: '#/components/schemas/EnsemblerStandardConfig_experiment_mappings'
           type: array
+        route_name_path:
+          pattern: ^\w*(?:\.\w+)*$
+          type: string
```

Comment:
This regex pattern here allows only period-delimited words of letters (or underscores) to be accepted as a valid `route_name_path`. Specifying this here also dispenses the need for manually crafting regex checks on the API for this field.